src/components/Shared/ClearGridPhyConfirmation.js

```diff
@@ -0,0 +1,21 @@
+import { ControlContext } from 'src/providers/ControlContext'
+import ConfirmationDialog from '../ConfirmationDialog'
+import { useContext } from 'react'
+
+const ClearGridPhyConfirmation = ({ window, onConfirm, fullScreen, open }) => {
+  const { platformLabels } = useContext(ControlContext)
+
+  return (
+    <ConfirmationDialog
+      open={open[0] ? open[0] : false}
```

Comment:
I prefer not to access object elements by index. Instead of passing `open: [true, {}] ` from formshell, it would be better to restructure it as a key-value object, like { flag: true, obj: {} }, to avoid relying on fixed indices for access.